### PR TITLE
Add the demScen dimension to EDGE-T inputs

### PR DESCRIPTION
**Pull request by Loisel**
# Purpose of this PR
In REMIND, other demand sectors use the so-called `demScen` parameter to distinguish different SSP/SDP dependent demand trajectories. With the next Version of EDGE-T, the output contains this dimension, so that EDGE-T scenarios can also be generated with specific `demScen`s. This PR enables REMIND to read in the modified outputs.

The PR also **removes the iterative script** which can be found in the `edgeTransport` repo from now on. This change has been merged and reverted previously in #919 and #924 .

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Refactoring
- [x] New feature 

## Checklist:

- [x] My code follows the coding etiquette
- [x] I have performed a self-review of my own code
- [ ] Changes are commented, particularly in hard-to-understand areas
- [ ] I have updated the in-code documentation
- [ ] The model compiles and runs successfully (`Rscript start.R -q`)

## Further information (optional):

* Test runs are here:
There are currently no test-runs, since there are is no new inputdata.
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: config/default.cfg [hunk 1]
```diff
@@ -25,7 +25,7 @@ cfg$description <- "REMIND run with default settings"
 cfg$regionmapping <- "config/regionmappingH12.csv"
 
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- 6.311
+cfg$inputRevision <- 6.3120023
```
Comment (Loisel) [inline config/default.cfg:28]:
```suggestion
cfg$inputRevision <- 6.3120025
```

File: config/default.cfg [hunk 2]
```diff
@@ -25,7 +25,7 @@ cfg$description <- "REMIND run with default settings"
 cfg$regionmapping <- "config/regionmappingH12.csv"
 
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- 6.311
+cfg$inputRevision <- 6.3120025
```
Comment (LaviniaBaumstark) [inline config/default.cfg:28]: nice input data version number :-)

File: core/sets.gms [hunk 3]
```diff
@@ -66,7 +66,11 @@ gdp_SSP2EU
 gdp_SSP3        "SSP3 slowGROWTH slowCONV"
 gdp_SSP4        "SSP4  medGROWTH mixedCONV"
 gdp_SSP5        "SSP5 fastGROWTH fastCONV"
-gdp_SSP2_lowEn   "SSP2 with low energy"
+gdp_SSP2_lowEn  "SSP2 with low energy"
+gdp_SSP2EU_NAV_ele
```
Comment (LaviniaBaumstark) [inline core/sets.gms:70]: could you please add a short description of the scenarios

**Review by LaviniaBaumstark — CHANGES_REQUESTED** [review]
* please delete the items not relevant to your PR from the list in the PR template
* I thing, we fist need a calibration at least for SSP2 for this new structure before you could merge it

**Comment by Loisel** [comment]
> * please delete the items not relevant to your PR from the list in the PR template
> 
>     * I thing, we fist need a calibration at least for SSP2 for this new structure before you could merge it

Calibration is running here: `/p/tmp/aloisdir/remind/output`
As a warning: I'll be on vacation from tomorrow on, so maybe someone else has to debug/take care. If you do not want me to merge the changes now, feel free to do it on your own. Relevant are the PRs to [mrremind](https://github.com/pik-piam/mrremind/pull/303) and to [edge-T](https://github.com/pik-piam/edgeTransport/pull/193)

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**